contracts/base/Escrow/EscrowStorage.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract EscrowStorage {
+    mapping(bytes32 => bytes) data;
+    IERC20 borrowedAsset;
+
+    function getBytes(string memory key) public view returns (bytes memory) {
```

Comment:
Here you are getting data for a given key. but when/how do you store the key/data?